python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -813,15 +817,10 @@ def makeCameraImage(self, camera, dataId, calibs):
         This is useful for judging the quality or getting an overview of
         the features of the calib.
 
-        This requires that the 'ccd name' is a tuple containing only the
-        detector ID.  If that is not the case, change CalibConfig.ccdKeys
-        or set CalibConfig.doCameraImage=False to disable this.
-
         @param camera  Camera object
         @param dataId  Data identifier for output
```

Comment:
With the change from tuple to `dataId` I think you should update this description